### PR TITLE
Added nf-test for cnvkit/genemetrics

### PR DESCRIPTION
**Pull request by GallVp**
Added nf-test for cnvkit/genemetrics with #6226

### CODE REVIEW

File: modules/nf-core/cnvkit/genemetrics/main.nf
Comment (marrip) [inline modules/nf-core/cnvkit/genemetrics/main.nf:1]: These appear empty for some reason.

**Comment by marrip** [comment]
Thank you for adding nf-test! The testing part looks fine, it is just that the module `main.nf` and `meta.yml` are marked as empty files.

File: modules/nf-core/cnvkit/genemetrics/main.nf
Comment (GallVp) [reply modules/nf-core/cnvkit/genemetrics/main.nf:1]: Thank you for the review. You are right. They appear empty. Strange! They are not empty though as you can click on the three dots and look at the file. The tests are also passing.

Comment (GallVp) [reply modules/nf-core/cnvkit/genemetrics/main.nf:1]: Actually, the empty files mean that nothing was changed. Only the permissions were changed from 755 to 644. I did not do it. It was part of the nf-test migration done by nf-core/tools.

**Review by marrip — APPROVED** [review]
lgtm!

**Pull request merged**